chart/k8gb/templates/operator.yaml

```diff
@@ -15,10 +15,6 @@ spec:
       labels:
         name: k8gb
     spec:
-      {{- if .Values.k8gb.hostAliases }}
-      hostAliases:
-        {{- toYaml .Values.k8gb.hostAliases | nindent 8 }}
-      {{- end }}
```

Comment:
Can't we leave this here and also in `values.yaml` as an alternative option? These still can co-exist, right? 0 support on the "makefile level", but leaving the option there would be a safe thing to do, imho

Reply:
I'd like to drop fall back from controller as well, right now it is quite confusing